.drone.yml

```diff
@@ -0,0 +1,116 @@
+pipeline:
+
+  build-docker-image:
+    image: docker:17.09.1
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+    commands:
+      - docker build -t hocs-management-ui .
+    when:
+      branch: [master, feature/HOCS-1022-create-build-pipeline]
```

Comment:
```suggestion
```
The HOCS-1022 branch will disappear soon, so you don't want it in the `.drone.yml` forever.
You probably want to always build the docker image on every push.

Reply:
Removed specific feature branch and added in dev branch which we'll be using from now on.